packages/amplify/amplify_flutter_android/android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -4,4 +4,4 @@ zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
 
 # (nydillon) Must be <=6.1.1 until this issue is resolved: https://github.com/gradle/gradle/issues/14132
```

Comment:
Double check this is not still an issue when running integration tests

Reply:
Thanks for calling this out. I'm keeping a couple of issues in mind that we put in temporary solutions earlier due to not able to upgrade gradle. 

I will do some more testing on the linter, then address relevant TODOs.